src/aws-s3/orb.yml

```diff
@@ -16,18 +18,23 @@ commands:
         default: "false"
         type: boolean
     steps:
-      - run:
+      - aws-cli/install
+      - aws-cli/configure:
+          aws_access_key_id: $AWS_ACCESS_KEY_ID
```

Comment:
Any reason why these are hard-coded in the `aws-s3` orb but parameters for the `aws-cli` orb? I think either approach is fine initially, but they should be consistent.

Reply:
I'll add them as defaults to the parameters in `aws-cli`